content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+ऑपरेटिंग सिस्टम नोड उपयोग करता है।
+क्यूबलेट इस जानकारी को नोड से इकट्ठा करता है और इसमें प्रकाशित करता है
+कुबेरनेट्स एपीआई।
+
+## दिल की धडकने
+
+कुबेरनेट्स नोड्स द्वारा भेजी गई दिल की धड़कन, आपके क्लस्टर को यह निर्धारित करने में मदद करती है
+प्रत्येक नोड की उपलब्धता, और विफलताओं का पता चलने पर कार्रवाई करना।
+
+नोड्स के लिए दिल की धड़कन के दो रूप होते हैं:
+
+* नोड के `.status` के अपडेट
+* [लीज](/docs/concepts/architecture/leases/) ऑब्जेक्ट्स
+  'क्यूब-नोड-लीज' के भीतर
+  {{< Glosary_tooltip term_id="namespace" text="namespace">}}.
+  प्रत्येक नोड में एक संबंधित लीज ऑब्जेक्ट होता है।
+
+नोड के `.status` के अपडेट की तुलना में, लीज़ एक हल्का संसाधन है।
+दिल की धड़कन के लिए पट्टों का उपयोग करने से इन अद्यतनों का प्रदर्शन प्रभाव कम हो जाता है
+बड़े समूहों के लिए।
+
+क्यूबलेट नोड्स की `.status` बनाने और अपडेट करने के लिए ज़िम्मेदार है,
+और उनके संबंधित पट्टों को अद्यतन करने के लिए।
+
+- स्थिति में परिवर्तन होने पर क्यूबलेट नोड के `.status` को अपडेट करता है
+  या यदि कॉन्फ़िगर किए गए अंतराल के लिए कोई अद्यतन नहीं किया गया है। डिफ़ॉल्ट अंतराल
+  नोड्स के लिए `.status` अपडेट के लिए 5 मिनट का समय है, जो 40 से बहुत अधिक है
+  अगम्य नोड्स के लिए दूसरा डिफ़ॉल्ट टाइमआउट।
+- क्यूबलेट हर 10 सेकंड में अपने लीज ऑब्जेक्ट को बनाता है और फिर अपडेट करता है
+  (डिफ़ॉल्ट अद्यतन अंतराल)। लीज अपडेट स्वतंत्र रूप से होते हैं
+  नोड के `.status` के अपडेट। यदि लीज अपडेट विफल हो जाता है, तो क्यूबलेट पुनः प्रयास करता है,
+  एक्सपोनेंशियल बैकऑफ़ का उपयोग करना जो 200 मिलीसेकंड से शुरू होता है और 7 सेकंड पर कैप किया जाता है।
+
+## नोड नियंत्रक
+
+नोड {{< Glosary_tooltip text="controller" term_id="controller" >}} एक है
+कुबेरनेट्स प्लेन घटक को नियंत्रित करता है जो नोड्स के विभिन्न पहलुओं का प्रबंधन करता है।
+
+नोड के जीवन में नोड नियंत्रक की कई भूमिकाएँ होती हैं। पहला असाइन कर रहा है
+सीआईडीआर पंजीकृत होने पर नोड को ब्लॉक करता है (यदि सीआईडीआर असाइनमेंट चालू है)।
+
+दूसरा नोड नियंत्रक की नोड्स की आंतरिक सूची को अद्यतित रख रहा है
+क्लाउड प्रदाता की उपलब्ध मशीनों की सूची। बादल में दौड़ते समय
+पर्यावरण और जब भी कोई नोड अस्वस्थ होता है, तो नोड नियंत्रक क्लाउड से पूछता है
+प्रदाता अगर उस नोड के लिए वीएम अभी भी उपलब्ध है। यदि नहीं, तो नोड
+नियंत्रक नोड को उसके नोड्स की सूची से हटा देता है।
+
+तीसरा नोड्स के स्वास्थ्य की निगरानी कर रहा है। नोड नियंत्रक है
+के लिए जिम्मेदार:
+
+- यदि कोई नोड पहुंच से बाहर हो जाता है, तो `तैयार` स्थिति को अपडेट करना
+  नोड के `.status` क्षेत्र में। इस मामले में नोड नियंत्रक सेट करता है
+  `तैयार` स्थिति `अज्ञात` के लिए।
+- यदि कोई नोड अगम्य रहता है: ट्रिगरिंग
+  [एपीआई-प्रारंभिक निष्कासन](/docs/concepts/scheduling-eviction/api-eviction/)
+  अगम्य नोड पर सभी पॉड्स के लिए। डिफ़ॉल्ट रूप से, नोड नियंत्रक
+  नोड को `अज्ञात` के रूप में चिह्नित करने और सबमिट करने के बीच 5 मिनट प्रतीक्षा करता है
+  पहला बेदखली अनुरोध।
+
+डिफ़ॉल्ट रूप से, नोड नियंत्रक प्रत्येक 5 सेकंड में प्रत्येक नोड की स्थिति की जाँच करता है।
+इस अवधि को `--नोड-मॉनिटर-पीरियड` फ्लैग का उपयोग करके कॉन्फ़िगर किया जा सकता है
+`क्यूब-नियंत्रक-प्रबंधक` घटक।
```

Comment:
[Upstream English content]
```
By default, the node controller checks the state of each node every 5 seconds.
This period can be configured using the `--node-monitor-period` flag on the
`kube-controller-manager` component.
```

You should not localize flags, parameters, and arguments into your language.
And IMO it would be great to preserve the component names in English as well.

```suggestion
डिफ़ॉल्ट रूप से, नोड नियंत्रक प्रत्येक 5 सेकंड में प्रत्येक नोड की स्थिति की जाँच करता है।
इस अवधि को `--node-monitor-period` फ्लैग का उपयोग करके कॉन्फ़िगर किया जा सकता है
`kube-controller-manager` घटक।
```